packages/atlas/src/views/studio/MyUploadsView/UploadStatus/UploadStatus.tsx

```diff
@@ -271,7 +276,13 @@ export const UploadStatus: FC<UploadStatusProps> = ({ isLast = false, asset, siz
           </FileStatusContainer>
           <FileInfo size={size}>
             <FileInfoType warning={isReconnecting && size === 'compact'}>
-              {isVideo ? <SvgActionVideoFile /> : <SvgActionImageFile />}
+              {asset.type === 'video' ? (
+                <SvgActionVideoFile />
+              ) : asset.type === 'subtitle' ? (
+                <SvgActionClosedCaptions />
+              ) : (
+                <SvgActionImageFile />
+              )}
```

Comment:
![image](https://user-images.githubusercontent.com/51168865/186603206-0b7490dd-6d9a-4833-909a-184210405d0e.png)
I'm not sure where we setting these values in the code, but I guess, this should be somewhere in this file. Shouldn't we put subtitles size on the next column(the third one)? Below sizes of the video file and image file?
**Suggestion** - maybe in the second column we could show the language or language iso?

Reply:
Good call, added subtitle language in the second column